src/state/slices/assetsSlice/selectors.ts

```diff
@@ -22,8 +22,9 @@ export const selectAssetById = createCachedSelector(
   (byId, assetId) => byId[assetId] || undefined,
 )((_state: ReduxState, assetId: AssetId | undefined): AssetId => assetId ?? 'undefined')
 
-export const selectAssetNameById = createSelector(selectAssetById, asset =>
-  asset ? asset.name : undefined,
+export const selectAssetNameById = createSelector(
+  selectAssetById,
+  (asset): string => asset?.name ?? '',
```

Comment:
I don't understand why this change was needed.

Reply:
changes the type signature of the function to be safer and more explicit - it was previously returning `string | undefined` and requiring view layer consumers to opt chain on the result. 

the selector should return a string. 

the selector is not used anywhere else in the code base, and was previously completely unused, so there is no risk of regressions.